app/controllers/assets_controller.rb

```diff
@@ -307,4 +315,52 @@ def asset_data_type(asset)
 
     'file'
   end
+
+  def create_edit_image_activity(asset, current_user, started_editing)
```

Comment:
This could be refactored either into a Service, or a method in `Asset` model, or some other utility class. We should keep controller lean.

Instead of passing `started_editing` as boolean, we could use symbols, type: :start_editing, type: finish_editing. Much more readable what's going on.